tests/synthesizer/data/loader.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+def get_audio_path(relative_path: str):
+    return os.path.join(os.path.dirname(__file__), relative_path)
```

Comment:
are we not able to use the loader in tests/data/loader.py?

Reply:
yep we can - in that case we'll need to move `fake_audio.mp3` out of `tests/synthesizer` and put it in `tests/streaming/data/fake_audio.mp3`